hoss/dimension_utilities.py

```diff
@@ -55,6 +59,25 @@ def is_index_subset(message: Message) -> bool:
     )
 
 
```

Comment:
Need to make it a more configurable way to override dimensions

Reply:
to be addressed as part of DAS-2241(var info) and DAS-2238 (HOSS)

Reply:
I guess the comment is not there anymore..

Reply:
I don't see the comment anymore.. :)

Reply:
closing this to clean up the review comments.